tests/candev/main.c

```diff
@@ -23,21 +23,25 @@
 
 #include <debug.h>
 #include <errno.h>
+#include <isrpipe.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <isrpipe.h>
 #include "shell.h"
 #include "can/device.h"
 
 #if IS_USED(MODULE_CAN_LINUX)
-
 #include <candev_linux.h>
-
 static candev_linux_t linux_dev;
 
+#elif defined(MODULE_MCP2515)
+#include "candev_mcp2515.h"
+#include "mcp2515_params.h"
+
+static candev_mcp2515_t mcp2515_dev = { 0 };
```

Comment:
AFAIK global variables are initalized to 0 anyway, otherwise you could use `memset` in the test to set to 0, would that work?

Reply:
Ah, yes. It's a global variable, so it will always be initialized to 0. 
I simply put it there out of the (good?) habit of always initializing variables to 0 when declaring them.

Not sure if this explains the CI error, but we'll know in a minute :)